release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2643,3 +2643,17 @@ link:https://access.redhat.com/solutions/6324771[{product-title} 4.8.11 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+// TODO: Add this to the end of the appropriate 4.8 z-stream RNs that this check is added with.
+// TODO: Do we need to include this notice in all future z-stream RN as well?
+[id="ocp-4-8-preparing-upgrade-next-release"]
+==== Preparing to upgrade to the next {product-title} release
+
+// TODO: replace "TODO" with appropriate z-stream, and is "check" the correct term?
+{product-title} 4.8.TODO introduced a check that will impact upgrading to the next {product-title} release, which is currently planned to be {product-title} 4.9. This is because Kubernetes 1.22, which {product-title} 4.9 is expected to use, removed a significant number of deprecated `v1beta1` APIs.
+
+This check requires an administrator to provide a manual acknowledgment before the cluster can be upgraded from {product-title} 4.8 to 4.9. This is to help prevent issues after upgrading to {product-title} 4.9, where APIs that have been removed are still in use by the cluster. Administrators must evaluate their cluster for any removed APIs in use and migrate them to use the appropriate new API version. After this is done, the administrator can provide the admin acknowledgment.
```

Comment:
nit: "admin" -> "admistrator"?  Or just drop it here, because "the administrator can provide the acknowledgement" already has the "who?" context close by.

Reply:
Yeah probably. Though I wouldn't mind hearing more input on whether we have a better term in general to use other than "admin acknowledgement" or "acknowledgment". It's a bit of a mouthful. It's fine if it's correct, but it's at least worth making sure that this is the best term to use.